src/components/FilterBar.jsx

```diff
@@ -86,5 +94,9 @@ export default function FilterBar(props) {
 FilterBar.propTypes = {
   changeSortBy: PropTypes.func.isRequired,
   changeSortFormat: PropTypes.func.isRequired,
-  amountFilterVals: PropTypes.objectOf(PropTypes.number).isRequired,
+  queryParams: PropTypes.objectOf(PropTypes.number),
+  onAmountFilterChange: PropTypes.func.isRequired,
```

Comment:
Hmm, this pattern will require passing in custom functions for each filter. I think a better pattern might be to accept the `clearQueryParam` and `setQueryParam` functions and then have the custom logic live here. `AmountFilter` is a detail of the implementation of `FilterBar` and it shouldn't really be assumed